content/md/supplier-onboarder/dwl-product-import-tpl.md

```diff
@@ -17,3 +18,19 @@ Here is how to get them:
 1. Choose your preferred format: one file per family or one file with all families, in `CSV` or `XLSX`
 
 This template file can easily be imported in your Onboarder to add many products at once.
+
+## File format
+The template contains 3 lines in the header:
+1. attribute code, it allows the Onboarder to read the file when importing it
+1. attribute labels, it is the name of the attribute given by the retailer
+1. requirement, it shows you what attributes are required for [product completion](update-products.html#the-attribute-panel)
```

Comment:
requirement status of the attribute: Shows you the status of the attribute (required, optional) for the [product completion](update-products.html#the-attribute-panel)